src/test/resources/environment-it-sfdc-mailchimp.json

```diff
@@ -0,0 +1,41 @@
+{
+  "apiKey": "abc123",
+
+  "crmPrimary": "salesforce",
+
+  "salesforce": {
+    "username": "brett@impactupgrade.com",
+    "password": "4mJA6UrqffKZEP35cWnrqxGmeZJGCi2IFHM2P5ag",
+
+    "fieldDefinitions": {
+      "emailOptOut": "HasOptedOutOfEmail",
+      "emailBounced": "",
+      "paymentGatewayName": "Payment_Gateway_Name__c",
+      "paymentGatewayTransactionId": "Payment_Gateway_Transaction_Id__c",
+      "paymentGatewayCustomerId": "Payment_Gateway_Customer_Id__c",
+      "paymentGatewaySubscriptionId": "Payment_Gateway_Subscription_Id__c"
+    },
+    "customQueryFields": {
+      "account": ["External_Reference__c"],
+      "campaign": [],
+      "contact": ["External_Reference__c", "HasOptedOutOfEmail"],
+      "donation": ["Payment_Gateway_Name__c", "Payment_Gateway_Transaction_Id__c", "Payment_Gateway_Customer_Id__c"],
+      "recurringDonation": ["Payment_Gateway_Name__c", "Payment_Gateway_Subscription_Id__c", "Payment_Gateway_Customer_Id__c"],
+      "user": []
+    }
+  },
+
+  "mailchimp": [
+    {
+      "secretKey": "secret-key-here",
```

Comment:
Sign us up for a free tier? Use something like team+it@impactupgrade.com for the account?

Reply:
Good point - but before I proceed with this part:
came across an issue with Mailchimp key commited to Github. Once it in GH - the key get's (almost) instantly revoked by Mailchimp (tried 3 times). 
We should add smth like a "key protection" mechanism in the code/Github secrets.
Have 3 possible fixes for this issue:
- Add Base64 encoding for all "secret keys" everywhere in the code (env.getConfig.getDecoded?)
- Use Base64 encoded version of the key in the resource file. Can put if statement in the code to decoded in case of IT test and use raw key otherwise 
- Use Github secret and then pass Mailchimp key in some way (env var?)

Please let me know which fix is the most suitable for you.
@brmeyer